common/reactive/src/main/java/io/helidon/common/reactive/SubmissionPublisher.java

```diff
@@ -30,13 +30,13 @@
 import reactor.core.scheduler.Schedulers;
 
 /**
- * A {@link Flow.Publisher} that asynchronously issues submitted
- * (non-null) items to current subscribers until it is closed.
+ * A {@link Flow.Publisher} that asynchronously issues submitted (non-null) items to current subscribers until it is closed.
  *
  * @param <T> the published item type
+ * @deprecated This class will be removed in the next major release.
```

Comment:
Please explain what to use instead/what is used instead.
Also please mark class as `@Deprecated`

Reply:
I'm not planning on a direct replacement, `OriginThreadPublisher` is the closest thing.
Will add `@Deprecated`.

Reply:
Fixed.